activities/Clock.activity/locale.ini

```diff
@@ -45,8 +45,8 @@ TutoSetTimeTitle=Set-Time Mode
 TutoSetTimeContent=Click this icon to switch to set-time mode. In this mode you could move the hands to display the hour you want
 TutoSetTimeGameTitle=Game Mode
 TutoSetTimeGameContent=Click this icon to switch to set-time-Game mode. In this mode, the time is displayed in number format and you have to move the hands to set it in the matching hour
-TutoSetSecondsTitle = Seconds Hand Hide.
-TutoSetSecondsContent= Click this icon to hide the second hand.
+TutoSetSecondsTitle = Seconds Hand Hide
+TutoSetSecondsContent= Click this icon to hide the second hand
```

Comment:
The button will be use not only to hide the second hand but also to show it again when it's hide.